integr8ly/integreatly-operator
# Update operator to watch all CRDs that it creates, and reconcile when they  update

Required a release as the installation type now only supports `AllNamespaces`. 

I have also updated the Makefile directives to run the operator in `AllNamespaces` mode, when running this locally (this is done by providing a namespace  of "").

To test this:
- Create a cluster
- Create the various CRD, roles, etc.
- Create an installation CR
- Run the operator 
- once the installation has finished succesfully, there should be very few logs output from the operator and it should not be reconciling every 10 seconds.
- Change a value in a CRD (e.g. a keycloakrealm) and see that this operator kicks of a reconcile.
